demisto_sdk/commands/init/templates/BaseIntegration/BaseIntegration.py

```diff
@@ -83,93 +79,80 @@ def test_module(client: Client) -> str:
     :rtype: ``str``
     """
 
-    message: str = ''
-    try:
-        # TODO: ADD HERE some code to test connectivity and authentication to your service.
-        # This  should validate all the inputs given in the integration configuration panel,
-        # either manually or by using an API that uses them.
-        message = 'ok'
-    except DemistoException as e:
-        if 'Forbidden' in str(e) or 'Authorization' in str(e):  # TODO: make sure you capture authentication errors
-            message = 'Authorization Error: make sure API Key is correctly set'
-        else:
-            raise e
-    return message
+    # TODO: ADD HERE some code to test connectivity and authentication to your service.
+    # This  should validate all the inputs given in the integration configuration panel,
+    # either manually or by using an API that uses them.
+    client.baseintegration_dummy('dummy', 10)  # No errors, the api is working
+    return 'ok'
 
 
 # TODO: REMOVE the following dummy command function
 def baseintegration_dummy_command(client: Client, args: Dict[str, Any]) -> CommandResults:
-
-    dummy = args.get('dummy', None)
-    if not dummy:
-        raise ValueError('dummy not specified')
+    # dummy is a required argument
+    dummy = args['dummy']
```

Comment:
Don't we want to refrain from using direct access as it is less safe? @Itay4 what do you think?

Reply:
I think that if an argument is required, we should is direct access as if it fails, it is a problem. Also, a much clearer code.
If it is not required, we should use the `.get` method.

Reply:
i think it would just be another thing for us to explain to contributions so i'd just use `.get()` for all to avoid confusion